### DISCUSSION TITLE
Welcome to Telegram-Members-Adder Discussions!

### DISCUSSION THREAD

**Discussion by saifalisew1508** [saifalisew1508/Telegram-Members-Adder#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SantMarcelo** [comment]
I'm making it work here, when I start adding users to my group it ends like this:

[i] Adding session ended

I didn't find what it is, can anyone help me?

**Comment by Surendra212** [comment]
I don't know how to add members 
Can you guide me please
